src/midiEditorCommands.cpp

```diff
@@ -500,4 +500,23 @@ void cmdFocusNearestMidiEvent(Command* command) {
 		}
 	}
 }
+
+static BOOL CALLBACK cbFindFilterWindow(HWND hWnd, LPARAM lParam) {
+	WCHAR szText[128];
+	if ((GetWindowTextW(hWnd, szText, 128) != 0) &&
+		!wcsncmp(szText, L"Filter Events", 13)) {
+		*((HWND *)lParam) = hWnd;
+		return false;
+	}
+	return true;
+}
+
+void cmdMidiFilterWindow(Command *command) {
+	HWND editor = MIDIEditor_GetActive(), filter = NULL;
+	MIDIEditor_OnCommand(editor, command->gaccel.accel.cmd);
+	// TODO: we could also check the command state was "off", to skip searching otherwise
+	EnumWindows(cbFindFilterWindow, (LPARAM)&filter);
+	if (filter && (filter != GetFocus()))
+		SetFocus(filter); // focus the window
```

Comment:
nit: Please brace this. I know it's only a single line block, but bracing even single lines prevents hard to trace mistakes in future. Also, I'm aware a lot of existing OSARA code doesn't do this; this is a change I've made to my coding style recently. :)

Reply:
Done